_includes/ee-linux-install-reuse.md

```diff
@@ -137,56 +137,58 @@ You only need to set up the repository once, after which you can install Docker
 
 {% elsif section == "install-using-yum-repo" %}
 
-There are currently two versions of Docker EE Engine available:
+For Docker Enterprise Edition 2.1, use Docker EE Engine 18.09. Installation instructions start below the note.
 
-* 18.03 - Use this version if you're only running Docker EE Engine.
-* 17.06 - Use this version if you're using Docker Enterprise Edition 2.0 (Docker
-Engine, UCP, and DTR).
+> ***NOTE:*** If you need to run Docker EE 2.0, please see the following instructions:
+> * [18.03](https://docs.docker.com/v18.03/ee/supported-platforms/) - Use this version if you're only running Docker EE Engine.
+> * [17.06](https://docs.docker.com/v17.06/engine/installation/) - Use this version if you're using Docker Enterprise 
+> Edition 2.0 (Docker Engine, UCP, and DTR).
 
-1. By default, Docker EE Engine 17.06 is installed. If you want to install the
-18.03 version run:
 
-    ```bash
-    sudo yum-config-manager --enable docker-ee-stable-18.03
-    ```
-
-2.  Install the latest patch release, or go to the next step to install a specific version:
+1.  Install the latest patch release, or go to the next step to install a specific version:
 
     ```bash
     $ sudo yum -y install docker-ee
     ```
 
     If prompted to accept the GPG key, verify that the fingerprint matches `{{ gpg-fingerprint }}`, and if so, accept it.
 
-3.  To install a _specific version_ of Docker EE (recommended in production), list versions and install:
+
+2.  To install a _specific version_ of Docker EE (recommended in production), list versions and install:
 
     a.  List and sort the versions available in your repo. This example sorts results by version number, highest to lowest, and is truncated:
 
     ```bash
     $ sudo yum list docker-ee  --showduplicates | sort -r
 
-    docker-ee.x86_64      {{ site.docker_ee_version }}.ee.2-1.el7.{{ linux-dist }}      docker-ee-stable-17.06
+    docker-ee.x86_64      {{ site.docker_ee_version }}.ee.2-1.el7.{{ linux-dist }}      docker-ee-stable-18.09
     ```
 
     The list returned depends on which repositories you enabled, and is specific to your version of {{ linux-dist-long }} (indicated by `.el7` in this example).
 
-    b.  Install a specific version by its **fully qualified package name** which is the package name (`docker-ee`) plus the version string (2nd column) up to the hyphen, for example: `docker-ee-17.06.1.ee.2`
+    b.  Install a specific version by its **fully qualified package name** which is the package name (`docker-ee`) plus the version string (2nd column) up to the hyphen, for example: `docker-ee-18.09.ee.2`
```

Comment:
The package example is actually wrong, as of `18.09` we no longer use the `.ee.X` naming scheme.

A more accurate example would be something like: `docker-ee-18.09.0`

Reply:
Fixed.